### ISSUE TITLE
Quick capture button is not toggleable

### ISSUE THREAD

**Issue by 2-4601** [WhisperSystems/TextSecure#3698]
I can turn on the quick capture mode by tapping the camera button.
But tapping it again does not dismiss the capture mode. Only system _Back_ works.

The accessibility(?) string says it should be toggleable:
`quick_attachment_drawer_toggle_description = "Toggle attachment drawer"`

- TS 2.23.0
 * Nexus 4, stock 5.1.1
 * Huawei U8800, 4.0.4

**Comment by McLoo** [comment]
Sorry,  dupe #3573 fixed with #3574

**Comment by 2-4601** [comment]
Nice! Thanks
